src/ORM/DataObject.php

```diff
@@ -289,6 +289,15 @@ class DataObject extends ViewableData implements DataObjectInterface, i18nEntity
      */
     private static $table_name = null;
 
+    /**
+     * Settings used by the FormScaffolder that scaffolds fields for getCMSFields()
+     */
+    private static array $scaffold_cmsfields_settings = [
```

Comment:
```suggestion
    private static array $scaffold_cms_fields_settings = [
```

Reply:
I think it's better as-is. But I really don't want to argue about it so I've made the change and will update all of its usages in the CMS 6 PRs. This is very subjective, though.